update.sh

```diff
@@ -9,67 +9,63 @@ if pidof -o %PPID -x "update.sh" > /dev/null; then
   exit
 fi
 
-update_target=$SATURN_HOME/update.sh
-
-echo -n "$(date -u) Checking for auto-update script ($update_target) updates... "
-
-if wget -O "$update_target.tmp" -T 10 -t 3 "https://raw.githubusercontent.com/filecoin-saturn/L1-node/main/update.sh" && [[ -s "$update_target.tmp" ]] && [ "$(stat -c %s "$update_target.tmp")" -ne "$(stat -c %s "$update_target")" ]
-then
-  mv -f "$update_target.tmp" "$update_target"
-  chmod +x "$update_target"
-  echo "updated $update_target script successfully!"
-  exit
-else
-  echo "$update_target script up to date"
-  rm -f "$update_target.tmp"
-fi
-
-run_target=$SATURN_HOME/run.sh
-
-echo -n "$(date -u) Checking for run script ($run_target) updates... "
-
-if wget -O "$run_target.tmp" -T 10 -t 3 "https://raw.githubusercontent.com/filecoin-saturn/L1-node/main/run.sh" && [[ -s "$run_target.tmp" ]] && [ "$(stat -c %s "$run_target.tmp")" -ne "$(stat -c %s "$run_target")" ]
-then
-  mv -f "$run_target.tmp" "$run_target"
-  chmod +x "$run_target"
-  echo "updated $run_target script successfully!"
-  exit
-else
-  echo "$run_target script up to date"
-  rm -f "$run_target.tmp"
-fi
-
-echo -n "$(date -u) Checking for Saturn $SATURN_NETWORK network L1 node updates... "
-
-out=$(sudo docker pull ghcr.io/filecoin-saturn/l1-node:$SATURN_NETWORK)
-
-if [[ $out != *"up to date"* ]]; then
-  echo "$(date -u) New Saturn $SATURN_NETWORK network L1 node version found!"
-
-  random_sleep="$(( RANDOM % 3600 ))"
-  echo "$(date -u) Waiting for $random_sleep seconds..."
+compose_file=$SATURN_HOME/docker-compose.yml
+env_file=$SATURN_HOME/.env
+
+echo "$(date -u) Starting migration to docker compose setup."
+echo "The auto-update script was deprecated to migrate to a docker compose setup. Updates are handled by a Watchtower container now."
+echo "The run script was deprecated too. To start or stop your node use 'docker compose -f $compose_file up -d' or 'docker compose -f $compose_file down' respectively."
+
+if [ -f "$compose_file" ]; then
+    echo "We have migrated to a docker compose setup to reduce risks caused by these update and run scripts. You can delete these scripts and use the Docker Compose workflow now."
+else 
```

Comment:
This is meant to prevent (security) issues where leftover cron jobs might execute this script every 5 minutes.
After running this update.sh once, if it worked it won't run anymore, not downloading nor running any random things with `sudo` anymore.